src/server.cpp

```diff
@@ -2922,6 +2930,19 @@ std::set<std::string> Server::getPlayerEffectivePrivs(const std::string &name)
 	return privs;
 }
 
+void Server::setPlayerFov(RemotePlayer *player, const f32 fov)
```

Comment:
Please use `player->setFov(fov); SendPlayerFov(player->getPeerId());` like it's done similar with other setter&send functions (`set_pos`, `set_hp`).